package.json

```diff
@@ -55,11 +80,24 @@
     "husky": "^8.0.0",
     "lint-staged": "^15.1.0",
     "prettier": "^3.1.0",
-    "ramda": "^0.29.1"
+    "rollup": "^4.4.1",
+    "rollup-plugin-analyzer": "^4.0.0",
+    "rollup-plugin-babel": "^4.4.0",
+    "rollup-plugin-cleaner": "^1.0.0",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "rollup-plugin-terser": "^7.0.2",
+    "rollup-swc-preserve-directives": "^0.6.0",
+    "yalc": "^1.0.0-pre.53"
   },
   "peerDependencies": {
-    "@bigbinary/neeto-cist": "latest",
     "@bigbinary/neeto-commons-frontend": "latest",
-    "ramda": "^0.29.1"
+    "@bigbinary/neeto-cist": "latest",
+    "mousetrap": "^1.6.5",
+    "mousetrap-global-bind": "^1.1.0",
```

Comment:
Why are these entries added in peerDependencies?
Why not bundle it along with neeto-hotkeys (since you have opted for serving a bundled package)?

Reply:
Please watch this: https://vimeo.com/885507664/9fe453ad52